packages/suite/src/hooks/suite/useCountdownTimer.ts

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState, useCallback, useMemo } from 'react';
+import intervalToDuration from 'date-fns/intervalToDuration';
+
+export const useCountdownTimer = (deadline: number | string) => {
+    const deadlineTimestamp = useMemo(() => new Date(deadline).getTime(), [deadline]);
```

Comment:
roundPhaseDeadline has type string | number, not sure if it can be narrowed to just number

Reply:
it does but I think the round now [has all deadlines](https://github.com/trezor/trezor-suite/blob/ddff6887cb712e28ed9f03628881c8f106df3978/packages/coinjoin/src/client/CoinjoinRound.ts#L112) as `number`, so we could either narrow down the type of `deadline` in `useCountdownTimer` to `number` or leave it flexible with string but it could lead to errors potentially if the string is wrong.

Reply:
ok, narrowed to number 7e350c535